File: uni_tutorials/ex5/ex5.py
# f(x) = 1 + x * (x + 1)
f = lambda x: 1 + x * (x + 1)

# g(x, y) = x + y^2
g = lambda x, y: x + y ** 2

# h(y, x) = x + y^2
h = lambda y, x: x + y ** 2

print(f(2))        # Wynik: 7
print(g(2, 3))     # Wynik: 11
print(h(3, 2))     # Wynik: 11

File: uni_tutorials/haskell_basics/ex5/ex5.js
// f(x) = 1 + x * (x + 1)
const f = x => 1 + x * (x + 1);

// g(x, y) = x + y^2
const g = (x, y) => x + y * y;

// h(y, x) = x + y^2
const h = (y, x) => x + y * y;

console.log(f(2));    // Wynik: 7
console.log(g(2, 3)); // Wynik: 11
console.log(h(3, 2)); // Wynik: 11